lldb/test/API/python_api/watchpoint/watchlocation/TestTargetWatchAddress.py

```diff
@@ -201,8 +201,15 @@ def test_watch_address_with_invalid_watch_size(self):
                 value.GetValueAsUnsigned(), 365, wp_opts, error
             )
             self.assertFalse(watchpoint)
-            self.expect(
-                error.GetCString(),
-                exe=False,
-                substrs=["Setting one of the watchpoint resources failed"],
-            )
+            if self.getPlatform() == "windows":
```

Comment:
I suspect this depends on whether you're using the lldb-server implementation (optional on windows, and the only option elsewhere) or the in process implementation (the default on windows when running locally). Maybe just make it accept both (with a regex) and add a comment.

Reply:
Fixed